trino/client.py

```diff
@@ -487,6 +487,9 @@ def __init__(
 
     @property
     def columns(self):
+        if self.query_id and not self.cancelled:
+            while not self._columns and not self.finished:
```

Comment:
should the loop include `not self.cancelled`?

Reply:
I just thought we can't cancel the query while doing other operator (get cursor description in this case)